forms/app/covid_trace_request.xml

```diff
@@ -0,0 +1,573 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <h:head>
+    <h:title>Trace Request form</h:title>
+    <model>
+      <itext>
+        <translation lang="en">
+          <text id="/covid_trace_request/contact_info/n_address:label">
+            <value>Address: <output value=" /covid_trace_request/contact_address "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info/n_case_id:label">
+            <value>Outbreak Case ID: <output value=" /covid_trace_request/outbreak_case_id "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info/n_other_names:label">
+            <value>Other Names: <output value=" /covid_trace_request/contact_other_names "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info/n_phone:label">
+            <value>Phone: <output value=" /covid_trace_request/contact_phone "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info/n_surname:label">
+            <value>Surname: <output value=" /covid_trace_request/contact_surname "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info/n_town:label">
+            <value>Town: <output value=" /covid_trace_request/contact_town "/></value>
+          </text>
+          <text id="/covid_trace_request/contact_info:label">
+            <value>Contact information</value>
+          </text>
+          <text id="/covid_trace_request/contacts/contact_repeat/contact_profile/address:label">
+            <value>Address</value>
+          </text>
+          <text id="/covid_trace_request/contacts/contact_repeat/contact_profile/age_in_years:label">
+            <value>Age in years</value>
```

Comment:
When i insert the age of the contact, when the contact doc is created it doesn't reflect show the age and when i tried to edit the doc the age field is empty

Reply:
We are asking for age only but not in the contact summary, age is based on date_of_birth. We have two options here:
1. Calculate the approximate date_of_birth field in the contact document. This should show the age automatically.
2. Add a field in contact summary that shows the actual age entered in the form.

Reply:
yes lets do 1. and apply something similar to how it is handled in the person create/edit form when you check the option to put the age.